service/service.go

```diff
@@ -81,6 +82,7 @@ func New(config Config) (*Service, error) {
 			SchemeBuilder: k8sclient.SchemeBuilder{
 				appv1alpha1.AddToScheme,
 				releasev1alpha1.AddToScheme,
+				capzv1alpha3.AddToScheme,
```

Comment:
Why is this only necessary for this type?

Reply:
`AWSClusters` are already exposed by `G8sClient().Infrastructurev1alpha2()` but `AzureClusters` use the upstream CAPZ type with `CtrlClient()`

Reply:
Weird - I wonder if that's intentional.